src/ripple/app/tx/impl/NFTokenAcceptOffer.cpp

```diff
@@ -27,6 +27,33 @@
 
 namespace ripple {
 
+STAmount
+NFTokenAcceptOffer::getAmountForValidation(
+    AccountID const& account,
+    STAmount const& amount)
+{
+    if (account != amount.getIssuer())
+        return amount;
+    auto ret = STAmount(amount);
+    ret.negate();
+    return ret;
+}
```

Comment:
I want to start by saying that the way you have this coded is just fine.  However, if you want to, you can make `getAmountForValidation()`  a non-member function that is local to just this implementation file.  You would do that because it reduces the visibility of the function just a hair.  You can make this change because...

- The method is `private`, so it's not needed outside of this implementation file and
- It does not use any of the member variables (which you had already noted by making it a `static` member).

If you choose make this change you'd:

- Remove the member function declaration from the header file and
- Change the declaration here to look like...

```
static STAmount
getAmountForValidation(
    AccountID const& account,
    STAmount const& amount)
```

The leading `static` is what makes it local to this file.  The linker will not make a file scope `static` function visible outside of the containing translation unit.

Note that, if you do this, then the implementation of `getAmountForValidation()` must appear in the file ahead of any place where it's called.  That's already true right now.  I just want to give you a heads up in case you do this same kind of thing in the future.

Whether or not you make this change is entirely up to you.